.ci/README.md

```diff
@@ -116,25 +116,24 @@ based on those labels. Possible labels are listed below:
 ### Running the CI locally
 
 The CI can run on a local VM using the following instructions:
-- Setup a Fedora VM
-  - Create a Fedora VM having at least 4G RAM and a 30GB disk.
+- Setup a Fedora/CentOS/CentOS-Stream VM
+  - Create a Fedora/CentOS/CentOS-Stream VM having at least 4G RAM and a 30GB disk.
   - Create a `sudo` user that does not require `sudo` password.
-  - Run the bash commands replacing the `fedora` user with the created one and the`CI_JOB` with the desired one:
+  - Run the bash commands replacing the `CI_JOB` with the desired one:
     ```sh
-    dnf install -y git vim make wget openssl driverctl pciutils
-    dnf remove -y qemu-guest-agent
-    wget https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq
-    sudo -H -u fedora bash -c 'mkdir -p ~/go/src/github.com/kata-containers'
-    sudo -H -u fedora bash -c 'cd ~/go/src/github.com/kata-containers && git clone https://github.com/kata-containers/tests.git'
-    sudo -H -u fedora bash -c 'echo export GOPATH=/home/fedora/go >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo export PATH=/usr/local/go/bin:\$GOPATH/bin:\$PATH >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo export USE_DOCKER=true >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo export CI_JOB="CRIO_K8S" >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo cd ~/go/src/github.com/kata-containers/tests >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo source .ci/ci_job_flags.sh >> ~/.bashrc'
-    grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"
+    sudo dnf install -y git vim make wget openssl driverctl pciutils
+    sudo dnf remove -y qemu-guest-agent zram-generator-defaults # the latter required only in Fedora 33+
+    sudo -H -u $USER bash -c 'mkdir -p ~/go/src/github.com/kata-containers'
+    sudo -H -u $USER bash -c 'cd ~/go/src/github.com/kata-containers && git clone https://github.com/kata-containers/tests.git'
+    sudo -H -u $USER bash -c 'echo export GOPATH=/home/$USER/go >> ~/.bashrc'
+    sudo -H -u $USER bash -c 'echo export PATH=/usr/local/go/bin:\$GOPATH/bin:\$PATH >> ~/.bashrc'
+    sudo -H -u $USER bash -c 'echo export CI_JOB="CRIO_K8S" >> ~/.bashrc'
+    sudo -H -u $USER bash -c 'echo export USE_PODMAN=true >> ~/.bashrc'
+    sudo -H -u $USER bash -c 'echo cd ~/go/src/github.com/kata-containers/tests >> ~/.bashrc'
+    sudo -H -u $USER bash -c 'echo source .ci/ci_job_flags.sh >> ~/.bashrc'
+    sudo grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0" # required only in Fedora
```

Comment:
I appreciate this is just updating what was already there, but a couple of comments:

- There are lots of commands here, so why not put them into a script and just say "run this script"? That way the script could perform additional error checking, etc.
- Again, I know this isn't strictly part of the PR but these commands should have a shell prompt prefix (see https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#code-blocks).

Reply:
Actually you can run it as script on all platforms, if the pkg is missing it will be ignored and Cgroupsv1 is already present for centos (currently) so it won't hurt, do you think it still worth condition those?

Oh, there is `"'sh` but it seems it's not interpreted as `'"bash`, I'll switch
Thanks!